git-send-email.perl

```diff
@@ -200,13 +204,14 @@ sub do_edit {
 			if (($? & 127) || ($? >> 8)) {
 				die("the editor exited uncleanly, aborting everything");
 			}
-		} @_;
+		} @args;
 	} else {
-		system('sh', '-c', $editor.' "$@"', $editor, @_);
+		system('sh', '-c', $editor.' "$@"', $editor, @args);
 		if (($? & 127) || ($? >> 8)) {
 			die("the editor exited uncleanly, aborting everything");
 		}
 	}
+	return;
```

Comment:
It's not bogus so much as it's superfluous.  The function not ending with an explicit 'return;' is the same as ending with a non-value statement or a 'return undef'.  PBP prefers the former.